src/redux/books/booksSlice.js

```diff
@@ -14,8 +14,8 @@ const bookSlice = createSlice({
     },
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229797891-0dc0eb1d-fff3-4169-ac30-4e4838fa49d6.png)

**[OPTIONAL]**

It would be very great if you make all your commit messages more meaningful and capitalize because I saw that some are not capitalized 😨 Kindly consider them making capitalized and meaningful to make your pull request professional 👍